app/src/main/java/org/jellyfin/androidtv/ui/search/SearchFragmentDelegate.kt

```diff
@@ -0,0 +1,56 @@
+package org.jellyfin.androidtv.ui.search
+
+import androidx.fragment.app.Fragment
+import androidx.leanback.widget.HeaderItem
+import androidx.leanback.widget.ListRow
+import androidx.leanback.widget.OnItemViewClickedListener
+import androidx.leanback.widget.OnItemViewSelectedListener
+import androidx.leanback.widget.Row
+import org.jellyfin.androidtv.constant.QueryType
+import org.jellyfin.androidtv.data.service.BackgroundService
+import org.jellyfin.androidtv.ui.itemhandling.BaseRowItem
+import org.jellyfin.androidtv.ui.itemhandling.ItemLauncher
+import org.jellyfin.androidtv.ui.itemhandling.ItemRowAdapter
+import org.jellyfin.androidtv.ui.presentation.CardPresenter
+import org.jellyfin.androidtv.ui.presentation.CustomListRowPresenter
+import org.jellyfin.androidtv.ui.presentation.MutableObjectAdapter
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class SearchFragmentDelegate(
+	private val fragment: Fragment
```

Comment:
You can use a context here instead of the fragment.

Reply:
Looks like ItemLauncher still requires an activity, I'll open a PR to fix that so you can rebase on top of master once that is merged